packages/editor/package.json

```diff
@@ -90,7 +90,7 @@
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.7.4",
     "@babel/plugin-proposal-optional-chaining": "^7.7.5",
     "@babel/plugin-syntax-dynamic-import": "^7.7.4",
-    "@babel/preset-env": "^7.11.5",
```

Comment:
Why this upgrade?

Reply:
To make the tests work. Had to provide RouteProvider in userCreateOrEditView.tsx. There was a related github issue which recommended to upgrade to get rid of the error. And it worked ;)